dotcom-rendering/src/web/components/SignInGate/gates/fake-social-variant.tsx

```diff
@@ -16,7 +16,14 @@ const SignInGateFakeSocial = React.lazy(() => {
 });
 
 export const signInGateComponent: SignInGateComponent = {
-	gate: ({ ophanComponentId, dismissGate, guUrl, signInUrl, abTest }) => (
```

Comment:
Yeah we don't the switch is only mentioned because of the way the type was defined. I've changed the type signature of the props so we don't need to explicitly pass it in in the other components